backend/curriculum_tracking/serializers.py

```diff
@@ -244,16 +249,16 @@ class Meta:
         fields = ["timestamp"]
 
 
-class AddReviewerUserSerializer(serializers.ModelSerializer):
+class SetDueTimeSerializer(serializers.ModelSerializer):
```

Comment:
Maybe a better name would include the word "bulk".  It would be good to be explicit aboput what this serialser is for. 

This should also be the serializer class associated with the new action.